superset/utils.py

```diff
@@ -668,33 +666,29 @@ def setup_cache(app, cache_config):
 
 def zlib_compress(data):
     """
-    Compress things in a py2/3 safe fashion
+    Compress things
     >>> json_str = '{"test": 1}'
     >>> blob = zlib_compress(json_str)
     """
-    if PY3K:
-        if isinstance(data, str):
-            return zlib.compress(bytes(data, 'utf-8'))
-        return zlib.compress(data)
+    if isinstance(data, str):
+        return zlib.compress(bytes(data, 'utf-8'))
     return zlib.compress(data)
 
 
 def zlib_decompress_to_string(blob):
     """
-    Decompress things to a string in a py2/3 safe fashion
+    Decompress things to a string
     >>> json_str = '{"test": 1}'
     >>> blob = zlib_compress(json_str)
     >>> got_str = zlib_decompress_to_string(blob)
     >>> got_str == json_str
     True
     """
-    if PY3K:
-        if isinstance(blob, bytes):
-            decompressed = zlib.decompress(blob)
-        else:
-            decompressed = zlib.decompress(bytes(blob, 'utf-8'))
-        return decompressed.decode('utf-8')
-    return zlib.decompress(blob)
+    if not isinstance(blob, bytes):
```

Comment:
Why reversing the if? The code does not look like equivalent to the old py3 version

Reply:
@timfasubaa please remove the `not`.